crates/hyperdrive-math/src/short/max.rs

```diff
@@ -68,7 +68,7 @@ impl State {
         // Get the spot price and the open share price. If the open share price
```

Comment:
```suggestion
        // Calculate the spot price and the open share price. If the open share price
```

Reply:
I found a bunch of these; I'll do a global search & fix them.